web3/utils/module_testing/parity_module.py

```diff
@@ -0,0 +1,21 @@
+class ParityModuleTest:
+
+    def test_trace_replay_transaction(self, web3, mined_txn_hash):
+        trace = web3.parity.traceReplayTransaction(mined_txn_hash)
+        print trace
```

Comment:
This `print` statement should be removed.  Can we assert *something* about the structure of this trace?  Even basic things like what type it has? or maybe some basics about it's schema based on what parity says it should return?

Bonus points if we can assert anything related to the `mined_txn_hash` such as the `to` address or something, but only it it's reasonably easy to do.

Reply:
Have added assert statements